### DISCUSSION TITLE
Can't select game

### DISCUSSION THREAD

**Discussion by loool090** [ebkr/r2modmanPlus#941] [Q&A]
I can't select game. What doing?
![r2modman_L4CeHJKVtM](https://user-images.githubusercontent.com/116916778/211890661-1583a393-e69d-4d40-bbaf-842b852e0b82.png)

**Comment by Tybernn** [comment]
https://github.com/ebkr/r2modmanPlus/wiki/Error:-White-or-blank-game-select-screen-on-startup
